### DISCUSSION TITLE
onFilterDropdownOpenChange 请问有大佬知道这个方法为什么必须异步才生效吗

### DISCUSSION THREAD

**Discussion by kyoonart** [ant-design/ant-design#37621] [General]
### What problem does this feature solve?

https://codesandbox.io/s/sd5t0k?file=/index.js onFilterDropdownOpenChange 请问有大佬知道这个方法为什么必须异步才生效吗

**Comment by kiner-tang** [comment]
不太清楚你的意思，这边直接同步调用也是可以筛选的呀
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/10286961/190882089-9e04b3ac-b5d4-4943-8dd2-88631c2bb15c.png">

**Comment by kyoonart** [comment]
这样输入框不会自动聚焦的  感觉应该是表头元素还没有渲染完成吧 @kiner-tang

**Comment by kiner-tang** [comment]
还是不太清楚你的需求是什么？
<img width="630" alt="image" src="https://user-images.githubusercontent.com/10286961/190896859-c3fcf441-c98e-4f3f-833d-80fd88a00938.png">
输入框有自动聚焦呀！或许你可以详细描述一下你期望的效果是怎样的，以及目前是怎样的
